src/program/snabbvmx/README

```diff
@@ -0,0 +1,7 @@
+Usage:
```

Comment:
As it happens with other Snabb programs that consist of more subprograms, the main one simply lists the available subprograms. "snabbvmx lwaftr" user help is more extensive. In any case, since I added a README.me, what SnabbVMX does has been documented there.